autogtp/README.md

```diff
@@ -6,18 +6,27 @@ the SGF and training data at the end of the game.
 
 # Requirements
 
-* Qt 5.3 or later with qmake
+* cmake
```

Comment:
CMake is a not a dependency for AutoGTP. It's a dependency if you follow the build instructions from the root README.